templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
```

Comment:
Consider providing more detailed error messages from the form's errors instead of a generic 'Invalid credentials' message. This can help users understand what went wrong (e.g., 'Username is required').